styles/Hero.module.css

```diff
@@ -25,11 +26,51 @@
 
 .hero__photo {
   border-radius: 16px;
+  width: 100%;
   height: 100%;
+  object-fit: cover;
 }
 
 .hero__icon {
   color: var(--icon-default);
   height: fit-content;
   width: fit-content;
+  padding-top: 16px;
+}
+
+/*responsive styling*/
+@media (max-width: 1023px) {
```

Comment:
Could we extract 1023px  743px into variables?

Reply:
Yes, I think we can. 
But we won't be able to reference them if we just define them as variables under :root in global.css. 
An appropriate way to make them variables might be using SASS.
There is a discussion related to this topic, you can find more details in it.
https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries